specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.3/examples/CopyModelAuthorization.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "body": {}
+  },
+  "responses": {
+    "201": {
+      "headers": {
+        "Location": "{endpoint}/formrecognizer/v2.1-preview.3/custom/models/{modelId}"
+      },
+      "body": {
+        "modelId": "{modelId}",
+        "accessToken": "{accessToken}",
+        "expirationDateTimeTicks": 637190189980000000
```

Comment:
We model this as an Int32 (seconds since Unix epoch) but this sample value is an Int64